### PR TITLE
Add login and register views. 

### LINKED ISSUE #26: Create the registration/login/logout pages views
Don't forget to use `JQuery` (According to the project's requirement - use it as much as you can)

### PR DESCRIPTION
**Pull request by barBen**

### CODE REVIEW

**Review by origranot — DISMISSED** [review]
Good job!

File: DGN/Views/Users/Register.cshtml [hunk 1]
```diff
@@ -4,75 +4,85 @@
...
+                            <input asp-for="Lastname" class="form-control" />
+                            <span asp-validation-for="Lastname" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="Birthday" class="control-label"></label>
+                            <input asp-for="Birthday" class="form-control" />
+                            <span asp-validation-for="Birthday" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="Role" class="control-label"></label>
```
Comment (ghost) [inline DGN/Views/Users/Register.cshtml]: you should remove the role, the user controller will do default - Client

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — DISMISSED** [review]

File: DGN/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -40,10 +40,10 @@
                     </ul>
                     <ul class="navbar-nav nav-menu ml-auto">
                         <li class="nav-item btn-item">
-                            <a href="#" class="nav-btn register-btn">Register</a>
+                            <a href="/Users/Register" class="nav-btn register-btn">Register</a>
```
Comment (bsuro10) [inline DGN/Views/Shared/_Layout.cshtml]: <a asp-controller="Users" asp-action="Register"></a>

Comment (bsuro10) [reply DGN/Views/Shared/_Layout.cshtml]: Change a tag to include asp-controller and asp-action

**Review by bsuro10 — DISMISSED** [review]

**Review by bsuro10 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**